staticcheck.conf

```diff
@@ -1 +1,16 @@
-initialisms = ["inherit", "CNI", "IPAM"]
+initialisms = ["ACL", "API", "ASCII",
+	"CNI", "CPU", "CSS",
+	"DNS",
+	"EOF",
+	"GID", "GUID",
+	"HTML", "HTTP", "HTTPS",
+	"ID", "inherit", "IP", "IPAM", "IPSec",
+	"JSON",
+	"NAT", "NATT",
+	"QPS",
+	"RAM", "RPC",
+	"SLA", "SMTP", "SQL", "SSH",
+	"TCP", "TLS", "TTL",
+	"UDP", "UI", "UID", "URI", "URL", "UTF8", "UUID",
+	"VM",
+	"XML", "XMPP", "XSRF", "XSS"]
```

Comment:
This still needs to be changed.

```suggestion
initialisms = ["inherit", "CNI", "IPAM", "NAT", "NATT"]
```

Reply:
To clarify, including `"inherit"` causes the defaults to be included automatically, without listing them. Thus to add a list of terms to the default list, one declares `["inherit"]` followed by the additional terms. (This applies to all the other PRs involving the default list of terms.)